cmake/ElektraCompiling.cmake

```diff
@@ -61,10 +61,27 @@ if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
 
 	if (ENABLE_DEBUG)
 		set (EXTRA_FLAGS "${EXTRA_FLAGS} -fsanitize=undefined -fsanitize=integer")
-		set (CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -fsanitize=undefined -fsanitize=integer")
+		set (CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -lubsan")
```

Comment:
Ahh, sorry, github did not show me enough context and I thought it would be a place for both gcc+clang.